libs/SmartStore/SmartStoreTests/SFSmartStoreWithExternalStorageTests.m

```diff
@@ -419,6 +419,9 @@ - (void)testExternalStorageIsNotEncryptedWhenDbIsNotEncrypted {
 
 - (void)testExternalStorageUpsertWithOneMBSizePayloadInRegression {
     NSInteger numberOfIterations = 500;
+    if ([[UIDevice currentDevice].systemVersion floatValue] < 9.3 ) {
+        numberOfIterations = 100;
+    }
```

Comment:
@kchitalia with this fix, the test will work for both ios9.2 and ios9.3, so this code is good.
Without this change, it will fail in ios9.2 because ios9.2 will hit memory issue (somehow ios9.3 won't)